tee-worker/ts-tests/integration-tests/di_vc.test.ts

```diff
@@ -0,0 +1,166 @@
+            requestIdentifier
+        );
+
+        const eventsPromise = subscribeToEventsWithExtHash(requestIdentifier, context);
+        const res = await sendRequestFromTrustedCall(
+            context.tee,
+            context.api,
+            context.mrEnclave,
+            teeShieldingKey,
+            setUserShieldingKeyCall
+        );
+        await assertIsInSidechainBlock('setUserShieldingKeyCall', res);
+
+        const events = await eventsPromise;
+        const userShieldingKeySetEvents = events
+            .map(({ event }) => event)
+            .filter(({ section, method }) => section === 'identityManagement' && method === 'UserShieldingKeySet');
+
+        await assertInitialIdGraphCreated(context, new PolkadotSigner(wallet), userShieldingKeySetEvents);
+    });
+
+    assertions.forEach((assertion) => {
+        step(`request vc ${Object.keys(assertion)[0]} (alice)`, async function () {
+            let currentNonce = (
+                await getSidechainNonce(context.tee, context.api, context.mrEnclave, teeShieldingKey, aliceSubject)
+            ).toNumber();
+            const getNextNonce = () => currentNonce++;
+            const nonce = getNextNonce();
+            const requestIdentifier = `0x${randomBytes(32).toString('hex')}`;
+            console.log(`request vc ${Object.keys(assertion)[0]} for alice ...`);
+            const eventsPromise = subscribeToEventsWithExtHash(requestIdentifier, context);
+
+            const requestVcCall = await createSignedTrustedCallRequestVc(
+                context.api,
+                context.mrEnclave,
+                context.api.createType('Index', nonce),
+                new PolkadotSigner(context.substrateWallet.alice),
+                aliceSubject,
+                context.api.createType('Assertion', assertion).toHex(),
+                requestIdentifier
+            );
+
+            const callValue = await sendRequestFromTrustedCall(
+                context.tee,
+                context.api,
+                context.mrEnclave,
+                teeShieldingKey,
+                requestVcCall
+            );
+
+            await assertIsInSidechainBlock(`${Object.keys(assertion)[0]} requestVcCall`, callValue);
+            const events = await eventsPromise;
+            const vcIssuedEvents = events
+                .map(({ event }) => event)
+                .filter(({ section, method }) => section === 'vcManagement' && method === 'VCIssued');
+
+            assert.equal(
+                vcIssuedEvents.length,
+                1,
+                `vcIssuedEvents.length != 1, please check the ${Object.keys(assertion)[0]} call`
+            );
+            const trustedOperationResponse = context.api.createType(
+                'TrustedOperationResponse',
+                callValue.value
+            ) as unknown as TrustedOperationResponse;
+            await assertVc(context, new PolkadotSigner(context.substrateWallet.alice), trustedOperationResponse.value);
```

Comment:
Here You should pass `aliceSubject` not `PolkadotSigner`

Reply:
emm...why we should pass `aliceSubject`,doesn't seem to have anything to do with the `LitentryPrimivesIdentity` type.

Reply:
Because you want to verify subject, not the signer (maybe they both are the same) but the VC is issued for the subject, not the signer.

Reply:
maybe just naming confusing... should change as:
`const subject = new PolkadotSigner(context.substrateWallet.alice)`
`await assertVc(context, subject, trustedOperationResponse.value)`;
:p

Reply:
Still I don't understand why you need to use `Signer` type to assert the subject of vc. It would be more convinient to use `LitentryPrimivesIdentity` instead.

Reply:
Updates:
checking subject instead of compare account with signer.